### DISCUSSION TITLE
Directus flows $NOW

### DISCUSSION THREAD

**Discussion by Fbuona** [directus/directus#19476] [Feature Requests]
### Summary

Hi,
I'm currently blocked with my Directus flows. 
Actually I need to compare a personnal DateTime with the actual date of the day. Reading the Directus documentation I found exactly what I need: the $NOW dynamic variable. But this is never returning me anything.

Here is the code I wrote : 

```json
{
    "$trigger": {
        "date": {
            "_eq": "{{$NOW}}"
        }
    }
}
```

I tried with and without the {{ }}, I'am using a wrong syntax ?
Is there any alternative I can use ?

Thanks for your help.

Fabio

### Motivation

-

### Detailed Design

-

### Requirements List

-

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request.

If you think you've ran into an issue, please [open a Bug Report](https://github.com/directus/directus/issues/new?assignees=&labels=&projects=&template=bug_report.yml). If you have a usage question, [Discord](https://directus.chat) is the platform we use for that 👍🏻

**Comment by Zack-Heisnberg** [comment]
Hi , create 

a run script  operation before your code call it time_now

module.exports = async function(data) {
	return new Date().toISOString();
}


then access it's value

{
    "filter": {
        "date": {
            "_lte": "{{time_now}}"
        }
    }
}

**Reply by Fbuona** [reply]
Hello,

Thanks a lot for your time and your answer.
I'll try it out.

For information I solved it using a script as well :
using Date.now();

Thanks again.

Fabio
